src/components/TutorialPage.js

```diff
@@ -1,34 +1,37 @@
 import styled from 'styled-components/macro'
-import data from '../data/content.json'
+import { useState } from 'react'
 import AudioButton from './AudioButton'
+import { ReactComponent as LotusNoFill } from '../images/lotus-small.svg'
+import { ReactComponent as LotusFill } from '../images/lotus-fill-small.svg'
+
+export default function TutorialPage({
+  headline,
+  title,
+  imagefile,
+  audiofile,
+  description1,
+  description2,
+  description3,
+}) {
+  const [isBookmarkt, setIsBookmarkt] = useState(false)
 
-export default function TutorialPage() {
   return (
-    <>
-      {data.map(
-        ({
-          headline,
-          title,
-          imagefile,
-          audiofile,
-          description1,
-          description2,
-          description3,
-          id,
-        }) => (
-          <TutorialWrapper key={id}>
-            {imagefile && <img src={imagefile} alt="" />}
-            <h2>{headline}</h2>
-            <h3>{title}</h3>
-            <p>{description1}</p>
-            {audiofile && <AudioButton audiofile={audiofile} />}
-            <p>{description2}</p>
-            <p>{description3}</p>
-          </TutorialWrapper>
-        )
-      )}
-    </>
+    <TutorialWrapper>
+      <BookmarkIcon onClick={() => toggleBookmark()}>
+        {isBookmarkt ? <LotusFill /> : <LotusNoFill />}
+      </BookmarkIcon>
+      <h2>{headline}</h2>
+      <h3>{title}</h3>
+      {imagefile && <img src={imagefile} alt="" />}
+      <p>{description1}</p>
+      {audiofile && <AudioButton audiofile={audiofile} />}
+      <p>{description2}</p>
+      <p>{description3}</p>
+    </TutorialWrapper>
   )
+  function toggleBookmark() {
+    setIsBookmarkt(!isBookmarkt)
```

Comment:
Do I understand it correctly, that each TutorialPage has it's own state which is either isBookmarked true or fals? You could think about introducing a custom hook, where you store isBookmarked for all tutorials "globally".